### PR TITLE
Update developer-setup.md

### PR DESCRIPTION
**Pull request by gabx**
For Archlinux user, best is to use the mattermost package from AUR

### CODE REVIEW

**Comment by jasonblais** [comment]
Thanks @gabx! Queuing for review

**Comment by jasonblais** [comment]
Actually, wondering if anyone in the community who uses Archlinux would be able to help with review?

**Comment by gabx** [comment]
I am using Archlinux and Fedora. What can I do for you?

File: source/developer/developer-setup.md [hunk 1]
```diff
@@ -82,6 +82,10 @@ Any issues? Please let us know on our forums at: http://forum.mattermost.org
 
 ### Archlinux ###
 
+_TIP_: There is a package in the [Arch User Repository](https://aur.archlinux.org/) called [Mattermost](https://aur.archlinux.org/packages/mattermost/). Follow the [AUR instructions](https://wiki.archlinux.org/index.php/Arch_User_Repository) to build and install.
```
Comment (it33) [inline source/developer/developer-setup.md]: Thanks for the addition! Seems helpful. 

Propose being explicit this is an unofficial package and sharing how to find support, e.g.: 

```
There is an unofficial package in the [Arch User Repository](https://aur.archlinux.org/) maintained by [mtorromeo](https://github.com/mtorromeo/), called [Mattermost](https://aur.archlinux.org/packages/mattermost/). Follow the [AUR instructions](https://wiki.archlinux.org/index.php/Arch_User_Repository) to build and install. For sharing feedback, troubleshooting and bug reports, please leave a comment at: https://aur.archlinux.org/packages/mattermost/
```

Alternatively we could direct people to https://pre-release.mattermost.com/core/channels/installers-and-images if @mtorromeo might be interested in fielding questions there. 

Just some ideas.

Comment (gabx) [inline source/developer/developer-setup.md]: I made a new PR  with more instructions as suggested: Update developer-setup.md #685

Sorry but I am not a github PR geek and couldn't find how to keep on this one.

**Comment by esethna** [comment]
Closing this PR as it is taken over by https://github.com/mattermost/docs/pull/685

**Pull request closed without merging**